power bank section updated, power bank image added, #71 (#78)
Capstone-Skynet/Capstone-Skynet.github.io: docs/design/3_1_7_computing_power.tex

Ultimately it was determined that the advantages of using separate power supplies, in particular, avoiding motor noise, warranted the extra weight and complexity.

\subsubsection{Custom vs Commercial Power Bank}
Initially a custom designed power supply system was intended to be used. It was to consist of a lithium-polymer or lithium-ion battery and a battery management system. The battery management system (seen in Figure \ref{powerdiag}) consists of two DC-DC converters (with an approximate efficiency of 94\%), a battery protection board, and a battery monitor. 

\begin{figure}[H]

> \centering

Reviewer: Reference image source?

\label{powerdiag}
\end{figure}

> DC-DC conversion is accomplished through use of a buck converter for the 5 V output and a buck-boost converter for the 12 V output.

Reviewer 2: ```suggestion
Using a custom designed power supply, DC-DC conversion would have been accomplished through use of a buck converter for the 5 V output and a buck-boost converter for the 12 V output.
```

We don't use the buck/boost converter now.

> After designing the system, an alternative commercial power bank was found which fits the project requirements. Said commercial power bank consists of 3 lithium-ion batteries with a combined capacity of 3000mAh in addition to a power management circuit. The power bank provides a 5V/2A USB output port and a 12V/3A output port which are compatible with the Raspberry Pi and the Zedboard respectively. The rated maximum current output from each port is sufficient to simultaneously supply both computation boards, and the battery capacity is rated to provide 3 hours and 45 minutes of operation on a single charge.

Reviewer 2: ```suggestion
After designing the system, an alternative commercial power bank was found which fits the project requirements. Said commercial power bank consists of 3 lithium-ion batteries with a combined capacity of 3000mAh in addition to a power management circuit. The power bank provides a 5V/2A USB output port and a 12V/3A output port which are compatible with the Raspberry Pi and the Zedboard respectively. The rated maximum current output from each port is sufficient to simultaneously power both computation boards, and the battery capacity is rated to provide 3 hours and 45 minutes of operation on a single charge.
```

\begin{figure}[H]
\centering
\includegraphics[width=12cm]{img/power_bank.png}
\caption{Commercial Power Bank}
\label{powerbank}
\end{figure}

> Ultimately, the commercial power bank was chosen over the custom-designed solution as the commercial system provides all of the requisite functionality at a lower cost and smaller form factor.

Reviewer: ```suggestion
Ultimately, we choose the commercial power bank for its lower cost and smaller form factor to fulfill the requirements.
```
